src/test/java/fi/tkgwf/ruuvi/config/ConfigTest.java

```diff
@@ -170,4 +170,26 @@ void testInfluxDbFieldFilter() {
         assertTrue(predicate.test("temperature")); // Allowed
         assertTrue(predicate.test("dewPoint")); // Allowed
     }
+
+    @Test
+    void testparseFilterMode() {
+
+        assertTrue(Config.isAllowedMAC("AB12CD34EF56"));
+        assertTrue(Config.isAllowedMAC("XX12CD34EF56"));
+        assertTrue(Config.isAllowedMAC("ABCDEFG"));
+        assertFalse(Config.isAllowedMAC(null));
+        
+        final Properties properties = new Properties();
+        properties.put("filter.mode", "named");
+        try {
+            Config.readConfigFromProperties(properties);
+        } catch (final IllegalStateException expected) {
+            fail("There should have been an exception: ruuvi-names.properties is empty.");
```

Comment:
isn't this wrong?

Reply:
yes - it should say it shouldnt have been an exception.

Reply:
in fact that try catch is misleading - ill remove it

Reply:
unless we want to refactor and make it possible to change the contents of TAG_NAMES ?

Reply:
Better test coverage is always better, but not strictly required. You can do the refactoring and add testing for that, if you want